### ISSUE TITLE
Resolver file to use

### ISSUE THREAD

**Issue by PoppinKaito** [opensanctions/opensanctions#260]
Hello quick question, the resolver file got deleted out of the repo but when running resolve it needs the resolver file as a reference. is it still viable to use the previous one that existed?

**Comment by pudo** [comment]
Yes, it's usable (see https://github.com/opensanctions/opensanctions/issues/246), but if you continue to make your own judgements on it, it'll be out of sync with opensanctions.org.
